### ISSUE TITLE
Theme Selection: Light or Dark Mode

### ISSUE THREAD

**Issue by NateSeymour** [InspiranzaDEV/zenenti#52]
# Requirements
- The theme selector should be accessible in the app's settings, allowing the user to pick between "Light" and "Dark" themes.
- The theme change should be applied instantly without needing to restart the app.
- The user's theme preference should persist even after closing and reopening the app.
- The app should work seamlessly in both themes, ensuring all texts, icons, and UI components are visible and accessible.
- The themes should meet accessibility standards, particularly regarding color contrast and readability.

# User Story
**As a** user, **I want** to choose between a light or dark theme for the app, **so that** I can adjust the app's appearance based on my visual preferences and comfort.

# Dependencies
- Depends on #35.
- Depends on #27.
- Depends on Settings Screen (pending issue).

**Comment by NateSeymour** [comment]
DEFERRED to v2.
